RunForrest/src/js/mustache.js

```diff
@@ -0,0 +1,20 @@
+var TEMPLATE = 
```

Comment:
No need to be a js file with a long string for this template. Just add the template as it is in a single file, and parse it with Mustache `Mustache.parse(mytemplate.mustache);`. Ask Kristal for any help.